File: 3371-harshad-number/harshad-number.js
/**
 * @param {number} x
 * @return {number}
 */
var sumOfTheDigitsOfHarshadNumber = function(x) {
    let sum=0
    let str=x.toString()
    for(s of str){
        sum+=Number(s)
    }
    if(x%sum==0){
        return sum
    }else{
        return -1
    }

    }

File: 3371-harshad-number/harshad-number.py
class Solution(object):
    def sumOfTheDigitsOfHarshadNumber(self, x):
        """
        :type x: int
        :rtype: int
        """
        sum=0
        s=str(x)
        for i in s:
            sum+=int(i)
        if x%sum==0:
            return sum
        return -1